docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -55,12 +56,35 @@
 \wss{You are free to modify this template.}
 
 \section{Introduction}
-
-\wss{You can include your definition of what a hazard is here.}
+This document outlines the hazard analysis of Sayyara. The definition of hazard is any property or
+condition in the system along with conditions in the environment that may cause harm or damage.
+This definition is from Nancy Leveson's work. The hazards for Sayyara include security and usage
```

Comment:
It would be nice if we could quote and cite Nancy Leveson directly (and professionally). idk how much work that is and how exactly to do it. The quote would come from the [STPA Handbook](https://psas.scripts.mit.edu/home/get_file.php?name=STPA_handbook.pdf).

> Definition: A _loss_ involves something of value to stakeholders. Losses may include a loss of human life or human injury, property damage, environmental pollution, loss of mission, loss of reputation, loss or leak of sensitive information, or any other loss that is unacceptable to the stakeholders.
> 
> Definition: A _hazard_ is a system state or set of conditions that, together with a particular set of worst-case environmental conditions, will lead to a loss.
>
> Definition: A _system_ is a set of components that act together as a whole to achieve some common goal, objective, or end. A system may contain subsystems and may also be part of a larger system.